scripts/generate-releases.sh

```diff
@@ -32,39 +31,39 @@ snapshot_list="$(kubectl get snapshot -l pac.test.appstudio.openshift.io/sha="${
 ')"
 
 validate_input() {
-    supported_ocp_number=$(find ".tekton" -maxdepth 1 -type f -name "operator-index-ocp-*-build.yaml" | wc -l )
-    snapshot_number=$(echo "$snapshot_list" | wc -l )
+    pipelines_count="$(find ".tekton" -maxdepth 1 -type f -name "operator-index-ocp-*-build.yaml" | wc -l )"
+    snapshots_count="$(echo "$snapshot_list" | wc -l )"
+    echo -e "found snapshots for \033[0;32m$COMMIT\033[0m commit in \033[0;32m$BRANCH\033[0m branch:" >&2
+    echo "$snapshot_list" >&2
 
-    if [ "$snapshot_number" -eq 0 ]; then
-        echo "ERROR: Could not find any Snapshots for the commit '${OPERATOR_INDEX_COMMIT}'. This must be a 40 character-long commit SHA. Default: currently checked out commit." >&2
+    if [[ "$snapshots_count" -eq 0 ]]; then
+        echo "ERROR: Could not find any Snapshots for the commit '${COMMIT}'." >&2
         exit 1
     fi
     if [[ "${ENVIRONMENT}" != "staging" && "${ENVIRONMENT}" != "prod" ]]; then
         echo "ERROR: ENVIRONMENT input must either be 'staging' or 'prod'." >&2
         exit 1
     fi
-    if [[ "${ENVIRONMENT}" == "prod" && "${OPERATOR_INDEX_BRANCH}" != "master" ]]; then
+    if [[ "${ENVIRONMENT}" == "prod" && "${BRANCH}" != "master" ]]; then
         echo "ERROR: prod release has to be done on master branch" >&2
         exit 1
     fi
-    if [[ "$snapshot_number" -ne "$supported_ocp_number" ]]; then
-        echo "snapshot list:" >&2
-        echo "$snapshot_list" >&2
-        echo "ERROR: The number of snapshots for $OPERATOR_INDEX_COMMIT in branch $OPERATOR_INDEX_BRANCH does not match the number of supported OCP versions ($supported_ocp_number)." >&2
+    if [[ "$snapshots_count" -ne "$pipelines_count" ]]; then
+        echo "ERROR: The number of snapshots for $COMMIT in branch $BRANCH does not match the number of supported OCP versions ($pipelines_count)." >&2
         exit 1
     fi
 }
 
 generate_release_resources() {
-    for d in $snapshot_list; do
+    for d in ${snapshot_list[@]}; do
```

Comment:
I think, quotes shouldn't hurt here
```suggestion
    for d in "${snapshot_list[@]}"; do
```

Reply:
double quots removes newlines from `snapshot_list` so I changed the implementation